hack/make/binary-client

```diff
@@ -2,7 +2,8 @@
 set -e
 
 (
-	export BINARY_SHORT_NAME='docker'
+	. ${MAKEDIR}/.binary_setup
```

Comment:
Can you make this match the `source` line below? (quotes and `source` vs `.`) :smile: